tools/webhook-apicoverage/resourcetree/rule.go

```diff
@@ -1,15 +1,33 @@
+/*
+Copyright 2018 The Knative Authors
```

Comment:
nit: do you mind replacing 2018 with 2019 in the files you changed, please? :)

Reply:
Changed.